src/types/wantlist/index.js

```diff
@@ -72,9 +97,16 @@ class Wantlist {
   }
 
   sortedEntries () {
+    // TODO: Figure out if this is an actual bug.
+    // @ts-expect-error - Property 'key' does not exist on type 'WantListEntry'
```

Comment:
@achingbrain do you have an idea what this meant to be, I'm guessing key used to be what cid is today, but I'm not sure. Nor I have a clue how this supposed to be sorted.

Reply:
Looks like a bug.  I think this is just supposed to be a list of wantlist entries sorted by string CID, rather than insertion order which would be the default - the `.key` property access is probably due to misreading the docs on the output of `Map.entries()`?

I'm not sure what the value of it being sorted is, other than the output being stable - it's probably better as a set TBH.

Something we should create an issue for and maybe address later.